dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -137,13 +162,17 @@ internal class OkHttpUploader : Uploader {
         internal const val HEADER_EVP_ORIGIN_VERSION = "DD-EVP-ORIGIN-VERSION"
         internal const val HEADER_REQUEST_ID = "DD-REQUEST-ID"
 
+        internal const val KEY_EVENT = "event"
+        internal const val KEY_JVM_MAPPING_FILE = "jvm_mapping_file"
```

Comment:
```suggestion
        internal const val KEY_JVM_MAPPING_FILE = "jvm_mapping"
```
The file name needs to be exactly `jvm_mapping` (the part name doesn't really matter), other than that LGTM !